example-widget-semantic-ui/src/PowerLevelsPage/PowerLevelsPage.test.tsx

```diff
@@ -219,13 +230,23 @@ describe('<PowerLevelsPage />', () => {
     });
     const demoteButton = screen.getByRole('button', { name: /demote/i });
 
-    expect(promoteButton).not.toBeDisabled();
-    expect(demoteButton).toBeDisabled();
+    await waitFor(() => {
+      expect(promoteButton).not.toBeDisabled();
+    });
+
+    await waitFor(() => {
```

Comment:
Another example where we can coll back the async stuff. If the promote button fulfils the test that means that rendering was done. The second `waitFor` should then no longer be required.

Reply:
this is correct in the semantic ui example but the mui example seems to still require the first `waitFor` after the `userEvent.click` to pass the test...

<img width="445" alt="Screenshot 2024-05-03 at 12 38 36" src="https://github.com/nordeck/matrix-widget-toolkit/assets/26399/9845a3c7-6d36-4c64-b81d-1f184ad6d339">